src/instanceMethods/_destroy.js

```diff
@@ -0,0 +1,31 @@
+import globalState from '../globalState.js'
+import privateProps from '../privateProps.js'
+import privateMethods from '../privateMethods.js'
+
+export function _destroy () {
+  const innerParams = privateProps.innerParams.get(this)
+  if (!innerParams) {
```

Comment:
Yup, I'm not sure how exactly to reproduce that scenario, but it'll happen if you comment out that `if` statement and run `yarn run cy:start`.

`cy:*` tasks stand for Cypress which is very promising testing runner. Right now, I'm using it for  development process because it's super-fast to re-run a particular set of tests. Also, it's used to generate the coverage report: https://github.com/sweetalert2/sweetalert2/blob/master/.github/workflows/coverage.yml

Reply:
I suppose that if the popup is closed by user interaction, `_destroy` is called and `innerParams` gets unset, and then when the next popup is fired, `_destroy` is called again for the previous popup which has already been destroyed (and thus `innerParams` has already been unset). 

Yeah I've checked Cypress out in the past and it really looks like a great solution, especially for end-to-end tests that need to simulate user interaction. Good to hear that it's working well here.

Reply:
> Good to hear that it's working well here.

Almost perfectly :) I reported the issue recently and waiting for it to be released so I can re-enable the `coverage` GitHub workflow: https://github.com/cypress-io/cypress/issues/6053